templates/named.conf.erb

```diff
@@ -14,5 +14,15 @@ options  {
 include "<%= scope.lookupvar('::dns::localzonepath') %>";
 <% end -%>
 
+<%- if scope.lookupvar('::dns::acl_hash') and !scope.lookupvar('::dns::acl_hash').empty? -%>
```

Comment:
Isn't this redundant? We know it's a hash since validate_hash checked that for us. If it's empty, the loop iterator will just do nothing.